types/express-http-proxy/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for express-http-proxy 1.5
+// Project: https://github.com/villadora/express-http-proxy#readme
+// Definitions by: ulrichb <https://github.com/ulrichb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+import { RequestHandler, Request } from "express";
+import { RequestOptions } from "http";
+
+export interface ProxyOptions {
+    proxyReqPathResolver?: (req: Request) => string;
+    proxyReqOptDecorator?: (proxyReqOpts: RequestOptions, srcReq: Request) => RequestOptions;
+}
+
+declare function proxy(host: string, options?: ProxyOptions): RequestHandler;
+
+export default proxy;
```

Comment:
`export default` only works when tsconfig contains `"esModuleInterop": true`. Some people won't have this set, so you should use Typescript's commonjs-specific syntax `export = proxy` and matching import syntax `import proxy = require('express-http-proxy')`.

Reply:
@sandersn Oh, forgot that. Fixed it in a follow up commit. Thanks!